orval-labs/orval
# feat(query): Add default mutation key

## Status



**READY**

## Description

Add a default mutation key for React Query mutation generation if none was provided.

Fix https://github.com/orval-labs/orval/issues/1249

## Todos


## Steps to Test or Reproduce

Generate a React Query mutation without supplied mutation key to check if default key is used. Generate with a supplied mutation key to check if that one is used instead.
